src/main/java/nl/tudelft/contextproject/model/level/util/RoomExitPoint.java

```diff
@@ -0,0 +1,23 @@
+package nl.tudelft.contextproject.model.level.util;
+
+/**
+ * Exit point of a room.
+ * 
+ * Used for determining if a door connection point has been used.
+ */
+public class RoomExitPoint extends DoorLocation {
```

Comment:
alright, was just wondering if that was a more efficient way of implementing it, but you seem to have taken that into account!